index.js

```diff
@@ -1,18 +1,25 @@
-const readline = require('readline');
+const readline = require("readline");
```

Comment:
Convention is to use single quotes around strings unless you need to escape them.

```js
const readline = require('readline');
```